assert/_diff.ts

```diff
@@ -151,37 +152,38 @@ export function diff<T>(A: T[], B: T[]): Array<DiffResult<T>> {
   }
 
   function createFP(
-    slide: FarthestPoint,
-    down: FarthestPoint,
+    slide: FarthestPoint | undefined,
+    down: FarthestPoint | undefined,
```

Comment:
```suggestion
    slide?: FarthestPoint,
    down?: FarthestPoint,
```

Reply:
Issues with this, see comment https://github.com/denoland/deno_std/pull/4045#issuecomment-1875372330